emperor/support_files/js/color-view-controller.js

```diff
@@ -221,34 +297,53 @@ define([
     });
   };
 
-  ColorViewController.Colormaps = ['discrete-coloring-qiime',
-  'discrete-coloring', 'OrRd', 'PuBu', 'BuPu', 'Oranges', 'BuGn', 'YlOrBr',
-  'YlGn', 'Reds', 'RdPu', 'Greens', 'YlGnBu', 'Purples', 'GnBu', 'Greys',
-  'YlOrRd', 'PuRd', 'Blues', 'PuBuGn', 'Spectral', 'RdYlGn', 'RdBu', 'PiYG',
-  'PRGn', 'RdYlBu', 'BrBG', 'RdGy', 'PuOr', 'Viridis'];
-  ColorViewController.ColormapNames = ['Classic QIIME Colors',
-  'Discrete Coloring (Colorbrewer)', 'Orange-Red', 'Purple-Blue',
-  'Blue-Purple', 'Oranges', 'Blue-Green', 'Yellow-Orange-Brown',
-  'Yellow-Green', 'Reds', 'Red-Purple', 'Greens', 'Yellow-Green-Blue',
-  'Purples', 'Green-Blue', 'Greys', 'Yellow-Orange-Red', 'Purple-Red', 'Blues',
-  'Purple-Blue-Green', 'Spectral', 'Red-Yellow-Green', 'Red-Blue',
-  'Pink-Yellow-Green', 'Pink-Red-Green', 'Red-Yellow-Blue', 'Brown-Blue-Green',
-  'Red-Grey', 'Purple-Orange', 'Viridis'];
-  ColorViewController._colorbrewerDiscrete = ["#8dd3c7", "#ffffb3", "#bebada",
-  "#fb8072", "#80b1d3", "#fdb462", "#b3de69", "#fccde5", "#d9d9d9", "#bc80bd",
-  "#ccebc5", "#ffed6f", /*first list ends here*/ "#a6cee3", "#1f78b4",
-  "#b2df8a", "#33a02c", "#fb9a99", "#e31a1c", "#fdbf6f", "#ff7f00", "#cab2d6",
-  "#6a3d9a", "#ffff99", "#b15928"];
+  ColorViewController.Colormaps = [
+    {id: 'discrete-coloring-qiime', name: 'Classic QIIME Colors', type: 'Discrete'},
```

Comment:
I've moved them to headers of each section as objects with `type` property. However, this means there are two different types of objects in this list, so that will need to be taken into account my anything else using this list.